utility/Database/createTables.sql

```diff
@@ -69,6 +69,7 @@ TRUNCATE TABLE GENRES;
 CREATE TABLE MOVIES (
     mov_id INTEGER PRIMARY KEY,
     title VARCHAR(50) NOT NULL,
+    overview VARCHAR(400),
```

Comment:
Varchar(400) can get filled easily in case of a long overview or if this section is used as a summary.
How about using VARCHAR(MAX) 

[Refer This](https://stackoverflow.com/questions/564755/sql-server-text-type-vs-varchar-data-type#:~:text=Use%20nvarchar(max)%2C%20varchar,and%20varbinary(max)%20instead.&text=TEXT%20is%20used%20for%20large,is%20stored%20out%20of%20row.&text=These%20length%20limitations%20do%20not,of%20row%2C%20just%20like%20TEXT%20.)

Reply:
Done.